lib/ramble/docs/getting_started.rst

```diff
@@ -12,32 +12,68 @@
 Getting Started
 ===============
 
+----------------
+What is Ramble?
+----------------
+
+Ramble stands for Reproducible And Measurable Benchmarks in a Layered Environment.
+
+Ramble is a multi-platform experimentation framework to increase exploration
+productivity and improve reproducibility. Ramble is capable of driving software
+installation, acquiring input files, configuring experiments, and extracting results.
+It works on Linux, macOS, and many supercomputers.
+
+Ramble can be used to configure a variety of experiments for applications. These
+can include anything from:
+
+* Scientific parameter sweeps
+* Performance focused scalaing studies
+* Compiler flag sweeps
+
 --------------------
 System Requirements
 --------------------
 
-Ramble's dependencies are listed within the top level
-requirements.txt file.
+Ramble requires Python 3.6.8 or greater and
+`Spack <https://spack.io/about/#install-spack>`_, which is used for managing
```

Comment:
Ramble doesn't *require* spack, so if this is written in the readme, we should change it.

Some applications within ramble require spack, but other ones (i.e. `hostname`) do not

Reply:
Edited to 'some applications also require Spack...' Does that sound OK as a way to prompt folks to install Spack?

Reply:
Yeah, that sounds good.